backend/runner/pubsub/testdata/java/subscriber/pom.xml

```diff
@@ -3,12 +3,12 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>xyz.block.ftl.examples</groupId>
     <artifactId>subscriber</artifactId>
-    <version>1.0-SNAPSHOT</version>
+    <version>0.433.1</version>
```

Comment:
Errrr...?

Reply:
oops, I really should have looked at what I was commiting...